src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IFORequirements.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo } from 'react'
+import {
+  Text,
+  Flex,
+  AccountIcon,
+  AccountFilledIcon,
+  TeamBattleIcon,
+  TrophyFillIcon,
+  Box,
+  useTooltip,
+  LinkExternal,
+} from '@pancakeswap/uikit'
+import { useWeb3React } from '@web3-react/core'
+import { formatBigNumber } from 'utils/formatBalance'
+
+const NotOkNFT = () => (
+  <>
+    Set{' '}
```

Comment:
Can we first translate the full text then split it with 'Pancake Squad NFT' (that won't be translated) then use split text array to construct. In that way we will have more translation friendly text

Reply:
It's a good idea. I will implement it when I add translation